emcc.py

```diff
@@ -1726,6 +1726,8 @@ def include_and_export(name):
 
     if 'address' in sanitize:
       shared.Settings.USE_ASAN = 1
+      if not shared.Settings.UBSAN_RUNTIME:
+        shared.Settings.UBSAN_RUNTIME = 2
```

Comment:
maybe a comment? i'm not sure why this makes sense to do.

Reply:
I actually don't know why its needed, I'm just preserving the behaviour that was previously in system_libs.py where USE_ASAN implies ubsan.   The only difference is that by doing it this way I prevent two different versions of ubsan from being included at the same time.